stdlib/public/stubs/Stubs.cpp

```diff
@@ -140,12 +140,21 @@ static int swift_snprintf_l(char *Str, size_t StrSize, locale_t Locale,
 #endif
 
 template <typename T>
+static int swift_floatingPointToStringPrecision(bool Debug) {
```

Comment:
Personally, I find that much clearer, but I definitely don't feel strongly enough to argue for it.  If you're happy with it as is, great.